app/controllers/coronavirus_form/refer_from_other_app_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::ReferFromOtherAppController < ApplicationController
+  def from_find_support_app
```

Comment:
This is a good solution 👍

If one app changes we might need to update the other. If we see this happening, we could rename the methods / controller to be more generic, eg

```
class CoronavirusForm::SkipQuestionsController < ApplicationController
  def skip_questions
```

`skip_questions` would then populate the session given some params and redirect to the correct URL. I don't think that's necessary to do now though.